linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -51,6 +51,8 @@
 #define DEFAULT_QDISC_MAX_SIZE 50
 #define DEFAULT_QDISC_ENABLE_SIZE 10
 
+#define BLOCKING_FLOW 1
```

Comment:
1) probably we need to add the blocking property to the shim_eth_flow struct then, but usually the app is an IPCP and this is fine (for now)
2) it's not needed in shim-hv, but we can define it anywhere you like.